src/converters/typescript/interfaces.ts.template

```diff
@@ -25,7 +35,26 @@ export interface TypographyToken {
   fontWeight: number;
 }
 
+export interface FocusToken {
+  content?: string;
+  position?: string;
+  pointerEvents?: string;
+  top?: ValueUnit;
+  right?: ValueUnit;
+  bottom?: ValueUnit
+  left?: ValueUnit,
+  borderRadius?: string;
+  backgroundColor?: string;
+  border?: string;
+  boxSizing?: string;
+  boxShadow?: string;
+  zIndex?: string;
+  outline?: string;
+  wrapper?: string,
+  after?: string,
+}
+
 export interface ValueUnit {
   value: number | string;
-  unit: string | null;
+  unit?: string | null;
```

Comment:
Now `unit` might be string, null or **undefined**.
What would be the case for the new addition?

Reply:
I changed it like this because I wanted to use this interface with gradients, transitions and shadows but I didn't want to write the token JSON like this 

```json
"highlightBaseToHighlightDark1": {
  "value": {
    "value": "linear-gradient(0deg, {colors.highlightBase} 0%, {colors.highlightDark1} 100%)"
    "unit": null
},
```

When `unit` can be undefined it simplifies the usecase

Reply:
Thanks for the explanation, makes sense.